packages/wonder-blocks-form/src/components/__docs__/checkbox-accessibility.stories.mdx

```diff
@@ -0,0 +1,137 @@
+import {Meta, Story, Canvas} from "@storybook/addon-docs";
+import {StyleSheet} from "aphrodite";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import {Checkbox} from "@khanacademy/wonder-blocks-form";
+import {LabelSmall} from "@khanacademy/wonder-blocks-typography";
+
+<Meta
+    title="Form/Checkbox/Accessibility"
+    component={Checkbox}
+    parameters={{
+        previewTabs: {
+            canvas: {hidden: true},
+        },
+        viewMode: "docs",
+        chromatic: {
+            // Disables chromatic testing for these stories.
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+export const ErrorTemplate = (args) => {
+    const [checked, setChecked] = React.useState(false);
+    const errorState = !checked;
+    return (
+        <View>
+            <Checkbox
+                checked={checked}
+                onChange={setChecked}
+                error={errorState}
+                aria-describedby={errorState && "error-message"}
+                aria-required={true}
+                {...args}
+            />
+            {errorState && (
+                <LabelSmall style={styles.error} id="error-message">
+                    You must agree to the terms to continue
+                </LabelSmall>
+            )}
+        </View>
+    );
+};
+
+export const DisabledTemplate = (args) => {
+    const [checked, setChecked] = React.useState(false);
+    const errorState = !checked;
+    return (
+        <Checkbox
+            checked={checked}
+            onChange={setChecked}
+            label="Some setting"
+            description="You do not have permission to change this setting"
+            disabled={true}
+        />
+    );
+};
+
+## Accessibility
+
+### ARIA
+
+`Checkbox` can take in all ARIA props defined in Wonder Blocks Core types.
+
+Elements with role `"checkbox"` can have an `aria-checked` property that
+exposes the checked state to assistive technology. The dev does not have
+to worry about this because the Wonder Blocks Checkbox component is an
+`input` element with type `"checkbox"`, as this has built-in semantics and
+does not require ARIA.
+
+The current implementation of `Checkbox` uses `aria-describedby` with the
+label and description that may be passed in as props.
+
+See the Error section for information about `aria-invalid` and
+`aria-required`.
+
+### Error state
+
+The Wonder Blocks `Checkbox` component takes an `error` boolean prop. Setting
+this prop to true will set `aria-invalid` to true, and the color of the
+checkbox to red.
+
+When a form input is invalid, the dev should provide a reason for why
```

Comment:
What do you think about changing the term `dev` for something more generic (user or something like that)? I know this is mainly intended for devs 😅, but maybe we can leave this more open to other audiences :)